pkgs/by-name/ke/keylight-cli/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  babashka,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "keylight-cli";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "versality";
+    repo = "keylight-cli";
+    rev = "46ff8472f9544e16eea822bf274acf4a5cbedbe8";
+    sha256 = "sha256-gzTvMBa7JVckxLnltlR5XOj6BBbfPXZei7Wj3f1n4Kw=";
+  };
+
+  propagatedBuildInputs = [ babashka ];
```

```suggestion
  buildInputs = [ babashka ];
```
The path to babashka will be hardcoded in the script, so there is no need to propagate babashka.

---

done, adjusted.